test.js

```diff
@@ -26,4 +26,4 @@ check(Bezier.fromSVG("M 0 0 Q 0.5 1 1 0"));
 
 // plain and SVG cubic check
 check(new Bezier(0,0 , 0,1   , 1,1 , 1,0));
-check(Bezier.fromSVG("m 0 0 C 0 1 1 0 0 -1"));
+check(Bezier.fromSVG("m 0 0 c 0 1 1 1 1 0"));
```

Comment:
Because "m 0 0 C 0 1 1 0 0 -1" not correct SVG path data for this curve.
http://codepen.io/betula/pen/QbBzXM
I'am added separate test for relative notation of this curve in my second commit.